components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/AuthenticationContext.java

```diff
@@ -59,6 +62,16 @@ public void setUser(User user) {
         this.user = user;
     }
 
+    public Object getProperties(String key) {
```

Comment:
better to rename as getPropertyValue since we are only returning a single property value.